# request part
string inputString
---
# response part
string outputString
